transforms/code/ingest_2_parquet/kfp_ray/v1/ingest_2_parquet_wf.py

```diff
@@ -0,0 +1,178 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import kfp.compiler as compiler
+import kfp.components as comp
+import kfp.dsl as dsl
+from kfp_support.workflow_support.utils import (
+    ONE_HOUR_SEC,
+    ONE_WEEK_SEC,
+    ComponentUtils,
+)
+from data_processing.utils import GB
+
+
+# the name of the job script
+EXEC_SCRIPT_NAME: str = "ingest_2_parquet_transform.py"
+
+task_image = "quay.io/dataprep1/data-prep-kit/ingest_2_parquet:0.3.0"
```

Comment:
think this version will eventually also need .dev6 suffix

Reply:
Again, for the first version its good enough

Reply:
but now is the time,  do you think someone is going to fix this later?